dts/arm/nxp/nxp_imx95_m7.dtsi

```diff
@@ -526,52 +526,71 @@
 			};
 		};
 
-		netc: ethernet@4ca00000 {
-			reg = <0x4ca00000 0x1000000>;
-			interrupt-parent = <&irqsteer_master0>;
-			interrupts = <13 0 0>;
+		netc_blk_ctrl: netc-blk-ctrl@4cde0000 {
+			compatible = "nxp,imx-netc-blk-ctrl";
+			reg = <0x4cde0000 0x10000>,
+			      <0x4cdf0000 0x10000>,
+			      <0x4c810000 0x18>;
+			reg-names = "ierb", "prb", "netcmix";
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges;
 
-			emdio: mdio@4cce0000 {
-				compatible = "nxp,imx-netc-emdio";
-				reg = <0x4cce0000 0x1c44>;
-				clocks = <&scmi_clk IMX95_CLK_ENET>;
+			netc: ethernet {
+				compatible = "nxp,imx-netc";
+				interrupt-parent = <&irqsteer_master0>;
+				interrupts = <13 0 0>;
```

Comment:
what is the benefit of netc_blk_ctrl, when it only contains one note?
Or are there socs coming with more than one node?

Reply:
The NETC Domain Block Control (BLK_CTRL_NETCMIX) module consists of a group of registers to provide miscellaneous control and status for peripherals within the NETC Domain. These registers provide chip-level control outside the peripherals. So I added one [driver](https://github.com/zephyrproject-rtos/zephyr/pull/93294/files#diff-f68376ecfc67a40a633adab9ba63e9d4c22c00495109ac537c3af9b182b69ac2) to do some netc domain block control initialization, it should be done before netc modules initialization, so put netc modules inside of netc block.